templates/layout/sidebar_subscriptions.html.twig

```diff
@@ -1,30 +1,71 @@
-<div class="sidebar-subscriptions section {{ app.request.cookies.get(constant('App\\Controller\\User\\ThemeSettingsController::KBIN_GENERAL_SHOW_SUBSCRIPTIONS_IN_SEPARATE')) is not same as 'true' ? 'inline' : 'section--top' }}">
-    <h3>{{ 'subscriptions'|trans }}</h3>
-    <div class="subscription-list">
-        {% for magazine in magazines %}
-            <div class="subscription {{ openMagazine and openMagazine.name is same as magazine.name ? 'active' : '' }}">
-                <a href="/m/{{ magazine.name }}">
-
-                    {% if magazine.icon %}
-                        <img src="{{ asset(magazine.icon.filePath) | imagine_filter('avatar_thumb') }}" class="magazine-subscription-avatar" alt="{{ magazine.name }}'s icon" />
+<div class="sidebar-subscriptions {{ app.request.cookies.get(constant('App\\Controller\\User\\ThemeSettingsController::KBIN_GENERAL_SHOW_SUBSCRIPTIONS_IN_SEPARATE')) is not same as 'true' ? 'inline' : '' }}">

```

Comment:
Make this also a `aside` element instead of `div` just like the other sidebar. So it's easier for browsers /read mode to know it's related to the main content, but aside of the main.

Reply:
ow I see.. that is not possible or is it? Because you reuse the same layout for inside the aside :\

Reply:
I was even considering make the left sidebar more generic, so we can use it not only for subscriptions but other things...

Reply:
well firefox doesn't complain about an `aside` in an `aside` :D